thirtyinch/src/test/java/net/grandcentrix/thirtyinch/internal/TiFragmentPresenterDestroyTest.java

```diff
@@ -278,17 +285,43 @@ public void saviorFalse_retainFalse_backstackTrue_dontKeepActivitiesFalse_activi
         delegate.onDestroyView_beforeSuper();
 
         // And when the Activity is changing configurations.
+        hostingActivity.setChangingConfiguration(true);
         delegate.onSaveInstanceState_afterSuper(mSavedState);
         delegate.onDestroy_afterSuper();
 
         // Then assert that the Presenter is destroyed and not saved in the savior.
         assertThat(delegate.getPresenter().isDestroyed()).isTrue();
         assertThat(mSavior.presenterCount()).isEqualTo(0);
+
+        // When a new Activity gets recreated
+        final HostingActivity hostingActivity2 = new HostingActivity();
+
+        // And a new Fragment instance is created by the framework.
+        final TestPresenter presenter2 = new TestPresenter(new TiConfiguration.Builder()
+                .setUseStaticSaviorToRetain(false)
+                .setRetainPresenterEnabled(false)
+                .build());
+        final TiFragmentDelegate<TiPresenter<TiView>, TiView> delegate2
```

Comment:
This test does also test what happens if the back stack is popped after the Activity was recreated after a configuration change. So either the name of the test must be changed or we remove the additional code. What do you say, @passsy?